tests/run-centos-ci.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+# setting up virtual environment
+yum -y install epel-release
+yum install libselinux-python yum-utils \
```

Comment:
Why not combine this line and next one into one `yum install`

Reply:
combined.

Reply:
@Sheersha-jain looks like you forgot to combine, in the latest commit I see two lines:

# setting up virtual environment
yum -y install epel-release
yum install libselinux-python yum-utils \
  device-mapper-persistent-data \
  lvm2 gcc python-virtualenv

The second one does not have -y, and the script will prompt for y/n again.